hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/state/AbstractAuthnFailedErrorState.java

```diff
@@ -10,6 +10,7 @@
 public abstract class AbstractAuthnFailedErrorState extends AbstractState implements ResponsePreparedState {
 
     private static final long serialVersionUID = 8101005936409595481L;
+
```

Comment:
Is this a style thing?

Reply:
Yes, just to keep it consistent with other state controllers